src/main/java/hudson/plugins/fitnesse/FitnesseExecutor.java

```diff
@@ -68,6 +71,11 @@ public boolean execute(Launcher launcher,FilePath workspace, Run<?,?> build) thr
 				throw (InterruptedException) t;
 			return false;
 		} finally {
+			try {
+				killTest(getFitnessePage(build, false));
```

Comment:
I'm sorry, trying to understand what you are saying... do you mean I should put
try { killTest(getFitnessePage(build, false)); (...)
in the catch block instead of in the finally block?